docs/src/start/connect-azure.md

```diff
@@ -103,7 +103,7 @@ Here provide the configuration parameters that are required to create the device
 Upon successfully saved the configuration a new device has been created on the IoT Hub.
 The new device can be seen on the IoT Hub portal by navigating to **Explores** &rarr; **IoT Devices**.
 
-More info about registering a device can be found [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-authenticate-downstream-device?view=iotedge-2018-06)
+More info about registering a device can be found [here](https://learn.microsoft.com/en-us/azure/iot-edge/how-to-authenticate-downstream-device?view=iotedge-2018-06)
```

Comment:
Also this one redirected me to https://learn.microsoft.com/en-us/azure/iot-edge/how-to-authenticate-downstream-device?view=iotedge-1.5&viewFallbackFrom=iotedge-2018-06

Reply:
Though I don't think we need to include all of the query parameters in the URL, as some query parameters are applied due to browser localization, browser type (e.g. desktop or mobile).